src/client/utils/auto-increment-id-generator.js

```diff
@@ -0,0 +1,19 @@
+const MAX_SAFE_INTEGER = Number.MAX_SAFE_INTEGER || 9007199254740991;
+const MIN_SAFE_INTEGER = Number.MIN_SAFE_INTEGER || -9007199254740991;
+
+export default function createAutoIncrementIdGenerator () {
```

Comment:
`createAutoIncrementIdGenerator` is a bad name. 'Auto increment' phrase means that id will increment by itself. It is wrong because we increment value manually.
I propose to rename as `createIntegerIdGenerator`.